packages/vuetify/src/components/VVirtualScroll/VVirtualScroll.tsx

```diff
@@ -59,7 +59,10 @@ export const VVirtualScroll = genericComponent<new <T, Renderless extends boolea
 
   setup (props, { slots }) {
     const vm = getCurrentInstance('VVirtualScroll')
-    const { dimensionStyles } = useDimension(props)
+    const { dimensionStyles } = useDimension({
+      ...props,
+      height: props.height ?? window.innerHeight
+    })
```

Comment:
1. this isn't reactive
2. it can have any height, shouldn't be limited to the window

Reply:
Makes sense.

I also think It's user's responsibility to ensure VVirtualScroll is either placed in a fixed height container or has height prop defined